.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,24 +9,32 @@ assignees: ''
 
 ## Description
 
-__[ 1 paragraph concisely describing the bug ]__
+<!-- [ 1 paragraph concisely describing the bug ] -->
```

Comment:
The [ ] stuff isn't necessary for it to be a comment, but I don't know if people find that visually pleasing or not. I have no personal opinion one way the another.